src/net_processing.cpp

```diff
@@ -3838,7 +3838,7 @@ class CompareInvMempoolOrder
 bool PeerLogicValidation::SendMessages(CNode* pto)
```

Comment:
Style-nit in commit body of c581cc16bb5e490c0960ccf440de2d1e5f23c417:

Instead of a GitHub-branded `#1117`, which either needs online-access to GitHub or two local commands to show the referenced commit (`git log --grep='#1117 ' --merges`), what about simply saying `commit c581cc16bb`.

Same for the other one, but since it has two commits, maybe the merge commit can be mentioned `commit 82274c0`

Reply:
Thanks. Will refer directly to commit hashes in future.